line-sdk/src/main/java/com/linecorp/linesdk/openchat/ui/OpenChatInfoFragment.kt

```diff
@@ -103,34 +122,18 @@ class OpenChatInfoFragment : Fragment() {
                 val selectedCategory = viewModel.getSelectedCategory(which)
                 viewModel.category.value = selectedCategory
             }
-            .create()
             .show()
 
-    private fun setupDescription() {
-        descriptionEditText.addTextChangedListener(
-            FormatStringTextWatcher(::updateDescription, MAX_CHAT_DESCRIPTION_LENGTH)
-        )
-
-        descriptionEditText.setText(viewModel.description.value)
-    }
-
-    private fun setupName() {
-        nameEditText.addTextChangedListener(
-            FormatStringTextWatcher(::updateName, MAX_CHAT_NAME_LENGTH)
-        )
+    private fun setupDescription() =
+        descriptionEditText.setAfterTextChangedAction { description ->
+            viewModel.description.value = description
+        }
 
-        nameEditText.setText(viewModel.chatroomName.value)
-    }
+    private fun setupName() =
+        nameEditText.setAfterTextChangedAction { viewModel.chatroomName.value = it }
```

Comment:
Rename `it` to a clear name as https://github.com/line/line-sdk-android/pull/45/commits/b340c13e91dc997adc0078f0265d3345267151f5#diff-4a7cf3f05b85f5bcc5a5054ed564136eR128 did or refactor it as below.
```kotlin
private fun setupName() =
        nameEditText.setAfterTextChangedAction(viewModel.chatroomName::setValue)
```